DataCollection/src/Infrastructure/Persistence/DataCollectionContext.cs

```diff
@@ -166,6 +166,10 @@ public IEnumerable<WorkLogReportRecord> GetWorkLogReportRecords(int userId)
 
         public DbSet<Observer> Observers { get; set; }
 
+        public DbSet<Tide> Tides { get; set; }
+
+        public DbSet<Weather> Weather { get; set; }
```

Comment:
@kpunjani ... while testing I noticed that I already created these tables, but didn't create the datasets to support them. And then I realized that there are some missing *tables* (not just *DbSets*) that I need to add soon: habitat, activity, and maybe another that I'm forgetting right now.